src/optimal_image/dataclients/jpeg.rs

```diff
@@ -0,0 +1,39 @@
+extern crate dssim;
+extern crate image;
+extern crate imgref;
+extern crate lodepng;
+extern crate rgb;
+
+use dataclients::{ImageDataResult, Loader};
+use image::*;
+use rgb::RGBA;
+use std::path::Path;
+
+pub struct Jpeg {
```

Comment:
Can we add some basic documentation here?

Reply:
I will add some docs on all the functions in another PR, still have to familiarize with rust automatic docs :P